src/index.ts

```diff
@@ -211,7 +220,7 @@ export class GraphQLServer {
       const playgroundOptions = this.options.subscriptions
         ? {
             endpoint: this.options.endpoint,
-            subscriptionsEndpoint: this.options.subscriptions,
+            subscriptionsEndpoint: (this.options.subscriptions as SubscriptionServerOptions).path,
```

Comment:
Avoid type casts if possible. If you're construction an extra `subscriptionsOptions` object before, this should be possible.